pytensor/graph/fg.py

```diff
@@ -19,15 +19,30 @@
 )
 from pytensor.graph.basic import as_string as graph_as_string
 from pytensor.graph.features import AlreadyThere, Feature, ReplaceValidate
+from pytensor.graph.op import Op
 from pytensor.graph.utils import MetaObject, MissingInputError, TestValueError
 from pytensor.misc.ordered_set import OrderedSet
 
 
-if TYPE_CHECKING:
-    from pytensor.graph.op import Op
+ClientType = tuple[Apply, int]
 
-ApplyOrOutput = Apply | Literal["output"]
-ClientType = tuple[ApplyOrOutput, int]
+
+class Output(Op):
```

Comment:
It's quite surprising we don't have another class named `Output` anywhere else in the codebase